pkg/discovery/util/pod_util.go

```diff
@@ -89,16 +89,54 @@ func HasController(pod *api.Pod) bool {
 	return !hasNodeOwner(pod)
 }
 
+// extracts mirror pod prefix. Returns the prefix and extraction result.
+func GetMirrorPodPrefix(pod *api.Pod) (string, bool) {
+	if !isMirrorPod(pod) {
+		return "", false
+	}
+	return strings.Replace(pod.Name, pod.Spec.NodeName, "", 1), true
+}
+
+func GetMirrorPods(pods []*api.Pod) []*api.Pod {
+	glog.V(4).Info("Getting mirror pods.")
+	mirrorPods := []*api.Pod{}
+	for _, pod := range pods {
+		if isMirrorPod(pod) {
+			mirrorPods = append(mirrorPods, pod)
+		}
+	}
+	glog.V(4).Infof("Found %+v mirror pods", len(mirrorPods))
+	glog.V(5).Info(mirrorPods)
```

Comment:
V(4) -> V(3)
V(5) -> V(4) (we don't have V(5))

Reply:
Thank you! Good catch!